usaspending_api/download/helpers/elasticsearch_download_functions.py

```diff
@@ -66,7 +67,8 @@ def _get_download_ids(cls, filters: dict, size: int = 10000) -> QuerySet:
         search = cls._search_type().filter(filter_query).source([cls._source_field])
         ids = cls._get_download_ids_generator(search, size)
         flat_ids = list(itertools.chain.from_iterable(ids))
-        logger.info(f"Found {len(flat_ids)} {cls._source_field} based on filters")
+        write_to_log(message=f"Found {len(flat_ids)} {cls._source_field} based on filters")
```

Comment:
In order for this log message to get associated with the download_job, i think you'll need to amend the `_ElasticsearchDownload` class to have a constructor that takes in the `download_job` (model) object (model) a required constructor param.